packages/binaryninja.vm/binaryninja.vm.nuspec

```diff
@@ -2,14 +2,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>binaryninja.vm</id>
-    <version>4.2.6455.20250417</version>
+    <version>4.2.6455.20250716</version>
     <authors>Vector 35, Inc.</authors>
-    <description>Binary Ninja is an interactive decompiler, disassembler, debugger, and binary analysis platform built by reverse engineers, for reverse engineers.</description>
+    <description>Binary Ninja is an interactive decompiler, disassembler, debugger, and binary analysis platform.</description>
     <dependencies>
       <dependency id="common.vm" version="0.0.0.20250509" />
       <!-- vcbuildtools.vm installs signtool.exe needed by VM-Assert-Signature -->
       <dependency id="vcbuildtools.vm" />
     </dependencies>
     <tags>Disassemblers</tags>
+    <projectUrl>https://binary.ninja</projectUrl>
```

Comment:
Is this the free version or the commercial version? if we are being picky then we would need to specify it is the free version in the description as well as https://binary.ninja/free/?